WordPress/Classes/ViewRelated/Blog/My Site/MySiteViewController.swift

```diff
@@ -144,24 +150,6 @@ class MySiteViewController: UIViewController, NoResultsViewHost {
         NotificationCenter.default.addObserver(self, selector: #selector(showAddSelfHostedSite), name: .addSelfHosted, object: nil)
     }
 
-    private func showSitePicker(for blog: Blog) {
-        guard FeatureFlag.mySiteDashboard.enabled else {
-            return
-        }
-
-        let sitePickerViewController = SitePickerViewController(blog: blog, meScenePresenter: meScenePresenter)
-
-        sitePickerViewController.onBlogSwitched = { [weak self] in
-            self?.blogDetailsViewController?.showInitialDetailsForBlog()
-            self?.blogDetailsViewController?.tableView.reloadData()
-            self?.blogDetailsViewController?.preloadMetadata()
-        }
-
-        addChild(sitePickerViewController)
-        stackView.insertArrangedSubview(sitePickerViewController.view, at: 0)
-        sitePickerViewController.didMove(toParent: self)
-    }
-
     private func setupSegmentedControl() {
```

Comment:
FYI: we shouldn't show the segmented control for self-hosted sites. Not sure if this is planned for a future task, but it might be good to add it as a subtask so we won't forget about it. :)

Reply:
Created an issue! https://github.com/wordpress-mobile/WordPress-iOS/issues/17825